### ISSUE TITLE
The plugin does not support TLS v1.3?

### ISSUE THREAD

**Issue by timorpic** [shadowsocks/v2ray-plugin#57]
The plugin does not support TLS v1.3?

**Comment by Love4Taylor** [comment]
It depends on v2ray-core

**Comment by t1935111** [comment]
Duplicate of #82

**Comment by moralrebuild** [comment]
Any progress? v2ray-core already started to support TLS 1.3 from 4.18.1

**Comment by Mygod** [comment]
e5ada30
